schemas/2015-08-01/Microsoft.Network.json

```diff
@@ -294,42 +334,27 @@
                         "publicIPAddress": {
                             "$ref": "#/definitions/id"
                         }
-                    },
-                    "additionalProperties": false,
-                    "type": "object"
-                }
-            },
-            "type": "object"
-        },
-        "backendIPConfigurationProperties": {
-            "properties": {
-                "backendIPConfigurations": {
-                    "type": "array",
-                    "items": {
-                        "$ref": "#/definitions/id"
                     }
                 }
             },
-            "type": "object",
             "required": [
-                "backendIPConfigurations"
+                "name",
+                "properties"
             ]
         },
         "backendAddressPools": {
+            "type": "object",
             "properties": {
                 "name": {
                     "type": "string"
-                },
-                "properties": {
```

Comment:
So the "name" will be the only property of this object? VS will have a hard time to match a object to this definition because almost all objects here has a "name" property, if "name" is the only property, you can add ""additionalProperties": false," to help VS on matching.